mongodb/README.md

```diff
@@ -78,6 +78,14 @@ MongoDB stream source (`MongoDBSources.streamMongodb()`) watches the changes to
 documents in a collection and emits these changes as they arrive. Source uses 
 ( `ChangeStreamDocument.getClusterTime()` ) as native timestamp.
 
+Change stream is available for replica sets and sharded clusters that use
+WiredTiger storage engine and replica set protocol version 1 (pv1). Change
```

Comment:
1. it should be more clear this is actual a hard prerequisite and streaming sources will NOT work without it. 
2. it would be nice to have a link to MongoDB documentation.

Reply:
👍